samples/server/petstore/aspnetcore/src/IO.Swagger/Controllers/PetApi.cs

```diff
@@ -92,6 +92,8 @@ public virtual IActionResult FindPetsByStatus([FromQuery][Required()]List<string
             // return StatusCode(400);
 
             string exampleJson = null;
+            exampleJson = "<Pet>\n  <id>123456789</id>\n  <name>doggie</name>\n  <photoUrls>\n    <photoUrls>aeiou</photoUrls>\n  </photoUrls>\n  <tags>\n  </tags>\n  <status>aeiou</status>\n</Pet>";
```

Comment:
Any way to ONLY pick JSON examples here?

Reply:
@wing328  @jimschubert

Reply:
Probably a hack, but iterate responses in the generator code and remove them if they don't start with a "{". If you go that route, is suggest logging a warning any time an example is dropped by the generator in this way.

Mustache doesn't really have logic. It does allow custom filters, so maybe we could write a filter that will only write a value of it is valid json. An issue with that approach is that you can't pick "the first json example* like you could in The generator processing.

Reply:
There was a discussion to add an `isJson` tag before but no one seems to have time to implement that yet.

Reply:
why a tag rather than Mustache filter? just curious.

Reply:
oh. I get what you're saying. It would be weird to use a tag to get a single json object from ab list of mixed possible types, I think.